README.md

```diff
@@ -0,0 +1,96 @@
+# Overview
+
+A simple service that can provide random subhasitas (maxims). Sanskrit: सुभाषित, subhāṣita.
+
+# Getting Started
+
+## Environment setup
+
+**Dependencies**
+
+- Rust Compiler
+    - Visual Studio C++ Build tools (on Windows)
+    - gcc (on Linux)
+- Docker Engine
+
+**Windows instructions**
+
+```
+winget install --id Docker.DockerDesktop
+winget install --id Rustlang.Rustup
+
+# Restart terminal after that and verify this works:
+rustc --version
+
+# If Visual Studio is already installed, make sure it is up to date and reboot if necessary.
+```
+
+**Windows Subsystem for Linux (WSL) instructions**
```

Comment:
Instead of we adding the detailed steps, can we link to the official documentation? 
This will help if it gets updated in the future, we do not have to update our readme as well?

Reply:
It is useful to keep our copy of the instructions because there may be small differences in the setup each developer has. I have added the official links also.